Microsoft.Toolkit.Uwp.UI.Controls/InAppNotification/InAppNotification.Events.cs

```diff
@@ -54,15 +52,18 @@ private void OpenAnimationTimer_Tick(object sender, object e)
                 _openAnimationTimer.Stop();
                 Opened?.Invoke(this, EventArgs.Empty);
                 SetValue(AutomationProperties.NameProperty, StringExtensions.GetLocalized("WindowsCommunityToolkit_InAppNotification_NameProperty", "/Microsoft.Toolkit.Uwp.UI.Controls/Resources"));
-                peer = FrameworkElementAutomationPeer.CreatePeerForElement(ContentTemplateRoot);
-                if (Content?.GetType() == typeof(string))
+                if (ContentTemplateRoot != null)
```

Comment:
Ah, so `ContentTemplateRoot` was getting set to null somewhere when the page was navigated? Got it, the rest is just clean-up, eh?

Reply:
Yes. Once navigated away, ContentTemplateRoot was null.